src/OrchardCore.Modules/OrchardCore.Users/Views/UserButtons.cshtml

```diff
@@ -16,6 +20,10 @@
         <a class="btn btn-danger btn-sm" asp-action="Delete" asp-route-id="@user.UserId" data-url-af="RemoveUrl UnsafeUrl" >@T["Delete"]</a>
     }
     <a asp-action="EditPassword" asp-route-id="@user.UserId" class="btn btn-secondary btn-sm">@T["Edit Password"]</a>
+    if (isLockedOut)
+    {
+        <a asp-action="Unlock" asp-route-id="@user.Id" class="btn btn-danger btn-sm" data-url-af="RemoveUrl UnsafeUrl" data-title="@T["Unlock user"]" data-message="@T["Are you sure you want to unlock this user?"]">@T["Unlock"]</a>
+    }
```

Comment:
I think this view (and the driver) is the one that needs the editor (checkbox) for `IsLockoutEnabled`

It should be wrapped in a `if (hasManageUser)` check so that a user editing their own profile can't update that value.